openapi/3.1/README.adoc

```diff
@@ -0,0 +1,55 @@
+---
+title: "MicroProfile Open API 3.1 (under development)"
+date: 2022-06-28
+summary: "Release for MicroProfile 6.0"
+---
+MicroProfile Open API aims to provide a set of Java interfaces and programming models which allow Java developers to natively produce OpenAPI v3 documents from their JAX-RS applications.
```

Comment:
Can we still call it JAX-RS or better Jakarta RESTful Services?

Reply:
It is okay to still call JAX-RS as this acronym was mentioned many times in Jakarta RESTful Web Services.